tools/src/main/scala/org/apache/predictionio/tools/console/Console.scala

```diff
@@ -734,7 +734,7 @@ object Console extends Logging {
     }
   }
 
-  def getEngineInfo(jsonFile: File): EngineInfo = {
+  def getEngineInfo(jsonFile: File, engineDir: Option[String]): EngineInfo = {
```

Comment:
@chanlee514 should we just pass in a `String` here? It doesn't look like there would be a case for `None`. It also looks like `o.a.p.tools.Common.getEngineDirPath` has already taken care of getting the current working directory, so I think it should be safe to just drop using `user.dir` completely.